projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/experiment/Experiment.java

```diff
@@ -0,0 +1,340 @@
+   * {@link Optional#empty()} is returned.
+   *
+   * @param value        the value of the variable
+   * @param variableName the name of the variable
+   */
+  public Optional<VariableLevel> getLevel(String variableName,
+      ExperimentalValue value) {
+    return experimentalDesign.getLevel(variableName, value);
+  }
+
+
+  /**
+   * Adds a level to an experimental variable with the given name. A successful operation is
+   * indicated in the result, which can be verified via {@link Result#isSuccess()}.
+   * <p>
+   * <b>Note</b>: If a variable with the provided name is not defined in the design, the creation
+   * will fail with an {@link ExperimentalVariableNotDefinedException}. You can check via
+   * {@link Result#isFailure()} if this is the case.
+   *
+   * @param variableName a declarative and unique name for the variable
+   * @param level        the value to be added to the levels of that variable
+   * @return a {@link Result} object containing the added level value or declarative exceptions. The
+   * result will contain an {@link ExperimentalVariableNotDefinedException} if no variable with the
+   * provided name is defined in this design.
+   * @since 1.0.0
+   */
+  public Result<ExperimentalValue, Exception> addLevelToVariable(String variableName,
+      ExperimentalValue level) {
+    return experimentalDesign.addLevelToVariable(variableName, level);
+  }
+
+  /**
+   * @return the identifier of this experiment
+   */
+  public ExperimentId experimentId() {
+    return experimentId;
+  }
+
+
+  /**
+   * @return the collection of species in this experiment
+   */
+  public Collection<Species> getSpecies() {
+    return experimentalDesign.species.stream().toList();
+  }
+
+  /**
+   * @return the collection of specimens in this experiment
+   */
+  public Collection<Specimen> getSpecimens() {
+    return experimentalDesign.specimens.stream().toList();
+  }
+
+  /**
+   * @return the collection of analytes in this experiment
+   */
+  public Collection<Analyte> getAnalytes() {
+    return experimentalDesign.analytes.stream().toList();
+  }
+
+  /**
+   * Adds {@link Specimen}s to the experimental design. If the design contains more than one
+   * specimen, the {@link ExperimentalVariable} <code>specimen</code> is created and all specimens
+   * are added as levels.
+   *
+   * @param specimens The specimens to add to the experimental design
+   */
+  public void addSpecimens(Collection<Specimen> specimens) {
+    final String specimensVariableName = "specimen";
+
+    if (specimens.size() < 1) {
+      throw new IllegalArgumentException(
+          "Did not get any specimen to add.");
+    }
+    // only add specimen that are not present already
+    List<Specimen> newSpecimens = specimens.stream()
+        .filter(it -> !experimentalDesign.specimens.contains(it))
+        .toList();
+    experimentalDesign.specimens.addAll(newSpecimens);
+
+    if (experimentalDesign.specimens.size() > 1) {
```

Comment:
```suggestion
    if (experimentalDesign.specimens.isEmpty()) {
```

Reply:
or write a Util class to check with `public boolean hasElements(Collection<?> collection)` or similar to invert the expression with a good semantic

Reply:
I think the check is whether it is more than 1 so >= 2